web/themes/joinup/prototype/sass/components/_form-table.sass

```diff
@@ -23,3 +23,14 @@ thead th.form-table__header
 .form-table__cell--icon-container
   width: 40px
   padding: 0
+
+#password-policy-status
```

Comment:
Please re-consider using ids, try to use classes.
If you cannot use classes, please move this into drupal hacks.